docs/src/content/docs/reference/database/tables.md

```
@@ -0,0 +1,245 @@
+- Use `is` or `has` prefix for boolean fields
+
+### Examples
+
+**Good table names:**
+
+- `user`
+- `order`
+- `orderItem`
+- `userProfile`
+- `companySettings`
+
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)